app/javascript/store/index.js

```diff
@@ -14,5 +15,6 @@ export default new Vuex.Store({
     scores,
     statistics,
     admin,
+    froggo_team,
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Identifier 'froggo_team' is not in camel case  camelcase

Reply:
y acá jajajaj